### PR TITLE
Add Custom scheduler changes for all helm charts

### PR DESCRIPTION
**Pull request by Aman-Singh-ML**

### CODE REVIEW

**Review by Sabari-Arunkumar-ML — APPROVED** [review]

**Review by Thouqueer-Ahmed-ML — APPROVED** [review]

**Review by Nikhil-Kumar-ML — APPROVED** [review]

File: charts/sf-archival/Chart.yaml [hunk 1]
```diff
@@ -4,11 +4,11 @@ dependencies:
 - condition: global.hive.internal.enabled
   name: hive-and-hadoop
   repository: https://snappyflow.github.io/helm-charts
-  version: 0.1.2
+  version: 0.1.4
```
Comment (Venunadh-Veeralanka-ML) [inline charts/sf-archival/Chart.yaml:7]: why not 0.1.3?

**Review by Venunadh-Veeralanka-ML — CHANGES_REQUESTED** [review]
Please address the review comments.

File: charts/sf-datapath/sub_charts/autoscaling/sub_charts/prometheus/templates/server/deploy.yaml [hunk 2]
```diff
@@ -37,7 +37,7 @@ spec:
       priorityClassName: "{{ .Values.server.priorityClassName }}"
 {{- end }}
 {{- if .Values.server.schedulerName }}
```
[the remark this answers is not in the document]
Comment (Aman-Singh-ML) [reply charts/sf-datapath/sub_charts/autoscaling/sub_charts/prometheus/templates/server/deploy.yaml]: change done

File: charts/sf-datapath/sub_charts/autoscaling/sub_charts/prometheus/templates/server/sts.yaml [hunk 3]
```diff
@@ -37,7 +37,7 @@ spec:
       priorityClassName: "{{ .Values.server.priorityClassName }}"
 {{- end }}
 {{- if .Values.server.schedulerName }}
```
[the remark this answers is not in the document]
Comment (Aman-Singh-ML) [reply charts/sf-datapath/sub_charts/autoscaling/sub_charts/prometheus/templates/server/sts.yaml]: change done

File: charts/sf-archival/Chart.yaml (hunk 1 above)
Comment (Aman-Singh-ML) [reply charts/sf-archival/Chart.yaml]: change done

**Review by Venunadh-Veeralanka-ML — APPROVED** [review]
now changes are looking good

**Pull request merged**